src/pages/fiches/comprendre-le-mot-clef-this.md

```diff
@@ -0,0 +1,344 @@
+
+```js
+const fluffy = {
+  name: "Fluffy",
+  color: "Rainbow",
+  species: "Unicorn",
+
+  sayHi(style, humor) {
+    return `${this.name} the ${style} ${this.species} with the ${this.color} color and the ${humor} humor says hi !`
+  }
+}
+const fluffySayHi = fluffy.sayHi;
+
+fluffySayHi.call(fluffy, "pretty", "bright"); // mode strict ou non
+// 'Fluffy the pretty Unicorn with the Rainbow color and the bright says hi !'
+``` 
+
+- Dans un cas similaire, mais où vous souhaitez passer un tableau d'objets en argument, c'est ici alors qu'intervient l'usage de la méthode `.apply()`. La principale différence avec la méthode `.call()`, se situe dans la manière dont sont passés les paramètres des méthodes d'instances liées à l'objet. Ils le sont sous forme de tableau, mais le résultat sera identique au niveau de votre console :
+
+```js
+fluffySayHi.apply(fluffy, ["pretty", "bright"]) // mode strict ou non
+// 'Fluffy the pretty Unicorn with the Rainbow color and the bright says hi !'
+```
+
+- Le dernier cas de figure est préconisé lorsque vous désirez lié définitivement votre méthode d'instance au "this" de l'objet ciblé. Dans cette hypothèse, nous utiliserons la méthode `.bind()`. Au niveau des arguments passé en paramètre à cette méthode, `.bind()` fonctionne de la même manière que `.call()`. Au niveau de l'invocation de la méthode, celle-ci ne l'est pas immédiatemment, `.bind()`retourne une nouvelle fonction avec this et des arguments partiellement appliqués, qui peut être appelée ultérieurement avec des arguments supplémentaires.
+
+```js
+const fluffy = {
+  name: "Fluffy",
+  color: "Rainbow",
+  species: "Unicorn",
+
+  sayHi(style, humor) {
+    return `${this.name} the ${style} ${this.species} with the ${this.color} color and the ${humor} says hi !`;
+  }
+};
+
+const fluffySayHi = fluffy.sayHi.bind(fluffy);
+
+console.log(fluffySayHi("pretty", "bright"));  // mode strict ou non
+// 'Fluffy the pretty Unicorn with the Rainbow color and the bright says hi !'
+```
+
+> Retenons donc jusqu'ici que le comportement du this varie en fonction du contexte dans lequel il est invoqué et en fonction du mode strict ou mode non strict. Suivant ce cas de figures, il peut y avoir une perte du "this", donc du contexte. Il alors indiqué d'utiliser les méthodes `.call()`, `.apply()` && `.bind()` que nous venons de voir ci-dessus afin de récupérer ce qui fut perdu ^^ !
+
+A présent, si votre tasse de café est vide, que la motivation est toujours là et que vous n'avez pas perdu le fil, je vous propose de remplir votre tasse pour un tour d'horizon des mystères "liés" au "this"... 🌌 !
+
+#### This et les fonctions (fléchées ou non) :
```

Comment:
Tu peux changer le titre ici et mettre plutôt : 

## This et les fonctions fléchées